lib/presentation/features/home/bloc/home_state.dart

```diff
@@ -1,8 +1,10 @@
 import 'package:equatable/equatable.dart';
 import 'package:marvelapp_flutter/presentation/models/character_view_data.dart';
 
+enum Loading { loading, firstLoading, loaded }
```

Comment:
Interesting idea =) Names coulb be improved to reflect how it looks, for example 
enum Loading { fullScreen, inBottomRow, loaded }

Reply:
done